apps/ex_nvr/test/support/fixtures/devices_fixtures.ex

```diff
@@ -57,7 +57,9 @@ defmodule ExNVR.DevicesFixtures do
 
   defp build_stream_config(stream_config, "file") do
     Enum.into(stream_config, %{
-      location: valid_file_location()
+      filename: "big_buck.mp4",
+      temporary_path: valid_file_location(),
```

Comment:
If you are talking about `temporary_path`, I added it because in `device_fixtures/2` method, we call `Devices.create()` to create the device, which calls `copy_device_file/2`. So when we want to use the method for a file device, we will have an issue with the device file copy if the field is null, so that's why I set it value